### ISSUE TITLE
`reportImplicitOverride` not triggered in type stubs

### ISSUE THREAD

**Issue by Avasam** [microsoft/pyright#6158]
**Describe the bug**
As title says. I noticed this while testing for https://github.com/python/typeshed/issues/10871

**Code or Screenshots**
```python
class A:
    def foo(self): ...

class B(A):
    def foo(self): ...  # Only errors in `.py` files
```

![image](https://github.com/microsoft/pyright/assets/1350584/cf44caf8-5862-4c93-a2e6-ab5cc0275350)

Configs have `"reportImplicitOverride": "error"`

If this is intended behaviour, can it be documented?

**VS Code extension or command-line**
pyright@1.1.331
pylance v2023.10.21

**Comment by erictraut** [comment]
Good catch. I had previously disabled all override checks for stubs because older versions of typeshed violated these rules. It looks like these have been address because the latest version of stdlib stubs in typeshed do not violate any of these checks. I'll remove the special-casing for stubs.

**Comment by erictraut** [comment]
This is included in pyright 1.1.332, which I just published. It will also be included in a future release of pylance.
